src/olympia/bandwagon/views.py

```diff
@@ -148,7 +148,7 @@ def collection_listing(request, base=None):
     filter = get_filter(request, base)
     # Counts are hard to cache automatically, and accuracy for this
     # one is less important. Remember it for 5 minutes.
```

Comment:
Don't change it but this feels pretty stupid since cache-machine already caches counts for 5 minutes...